### DISCUSSION TITLE
A table that contains more than 300 fields. Do I need to split it into two tables?

### DISCUSSION THREAD

**Discussion by HWTsang** [apache/doris#30500] [A - General / Q&A]
I have the following fields: request_id, query_time, first_loan, apply_time, merchant_id, user_mobile, etc.  
There are also other fields, such as predict_value, score_level, score, result, model_score, etc.   
I have a total of 300+ fields. Should I store them in two separate tables and link them through a unique id, or should I just keep them in one table?  
Which method is better, or is there a better way to handle this? I’ve tried storing a lot of data in JSON before, but it led to memory overflow issues.

**Comment by LemonLiTree** [comment]
It can be stored in a unique table and mow is turned on. 300 fields are not many.
